File: Praktikum Pemrograman I Modul 3/Praktikum I Modul3 24-10-2023/C/PRAK301-2310817110015-AKHMAD_CHAIDAR_ANANDA.c
#include<stdio.h>

int main(){

    // Deklarasi Variabel
    int a, b, temp;

    // Input
    printf("");
    scanf("%d %d", &a, &b);

    // Pengkondisian
    if (a > b){
        temp = a; // Variabel temporary merupakan variabel sementara yang menyimpan nilai a
        a = b;
        b = temp;
        printf("%d %d", a, b);
    } else {
        printf("%d %d", a, b);
    }

    return 0;
}

File: Praktikum Pemrograman I Modul 3/Praktikum I Modul3 24-10-2023/Python/PRAK301-2310817110015-AKHMAD_CHAIDAR_ANANDA.py
# Input
a, b = map(int, input().split())

# Pengkondisian
if a > b:
    temp = a # Temp merupakan variabel sementara yang meyimpan nilai a
    a = b
    b = temp 
    print(f"{a:d} {b:d}") # Menggunakan format string
else:
    print(f"{a:d} {b:d}") # Menggunakan format string